test/e2e/specs/fse/fse__limited-global-styles.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @group calypso-pr
+ * @group gutenberg
+ */
+
+import {
+	DataHelper,
+	TestAccount,
+	FullSiteEditorPage,
+	SecretsManager,
+} from '@automattic/calypso-e2e';
+import { Browser, Page } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Site Editor: Limited Global Styles' ), function () {
+	let page: Page;
+	let fullSiteEditorPage: FullSiteEditorPage;
+	const credentials = SecretsManager.secrets.testAccounts.simpleSiteFreePlanUser;
+	const siteSlug = credentials.testSites?.primary?.url as string;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+
+		const testAccount = new TestAccount( 'simpleSiteFreePlanUser' );
+		await testAccount.authenticate( page );
+
+		fullSiteEditorPage = new FullSiteEditorPage( page );
+	} );
+
+	it( 'Visit the site editor', async function () {
+		await fullSiteEditorPage.visit( siteSlug );
+		await fullSiteEditorPage.prepareForInteraction( { leaveWithoutSaving: true } );
+	} );
+
+	it( 'Open site styles', async function () {
+		await fullSiteEditorPage.openSiteStyles();
+	} );
+
+	it( 'Select the "Try it out" option in the upgrade modal', async function () {
+		await fullSiteEditorPage.tryGlobalStyles();
+	} );
+
+	it( 'Update the global background color', async function () {
+		await fullSiteEditorPage.setGlobalColorStyle( 'Background', {
+			colorName: 'Primary',
+		} );
+	} );
+
+	it( 'Save the styles', async function () {
+		// On mobile, site styles is a popover panel that blocks the Save button.
+		// So let's always close site styles first to be safe. :)
+		await fullSiteEditorPage.closeSiteStyles();
+		await fullSiteEditorPage.save( { preSaveSelectors: [ '.wpcom-global-styles-notice' ] } );
+	} );
+
+	it( 'Reset styles to defaults', async function () {
+		await fullSiteEditorPage.openSiteStyles();
+		await fullSiteEditorPage.setGlobalColorStyle( 'Background', {
+			colorName: 'Primary',
+		} );
+		await fullSiteEditorPage.closeSiteStyles();
```

Comment:
Looking at the artefacts in TC it looks like it's getting stuck here. It's setting the global style but it isn't saving.

Reply:
Thanks! Seems that changing the background color wasn't working always (because the "Primary" color name is not robust enough), and following runs were failing to see the pre-save notice because the previous run didn't restore the original color.

I ended up adding methods to choose a different style variation which should be more robust and updated the flow to use that instead.

Reply:
Seems TC is still stuck here, but I don't understand why. The test is passing locally and I don't see anything obvious in the artifact video that might explain the issue – the save button should be clickable, but somehow it is not clicked. @worldomonation do you know what's going on?

Reply:
Alright, finally was able to figure out that the failure was caused by the cookie banner intercepting the click that hides the "Site updated" notification before styles are saved again.

I'm not sure if the cookie banner should be visible for E2E tests (asked here p1671789652736069-slack-C03N25JPCE4), but I added extra logic to dismiss it if present.